dockerfiles/docker-entrypoint.sh

```diff
@@ -121,11 +120,6 @@ function read_style() {
     slug_name=$1
     style_name=''
 
-    # This check is always true in CORGI and never true in webhosting pipeline.
-    if [[ -f $IO_BOOK/style ]]; then
-        style_name=$(cat $IO_BOOK/style)
-    fi
-
     if [[ ! $style_name || $style_name == 'default' ]]; then
```

Comment:
On line 121 `style_name=''`, `! $style_name` is true so I think we can remove this condition.  What do you think?

Reply:
Good catch, thanks! Updated